src/Confluent.Kafka/Producer.cs

```diff
@@ -323,7 +323,7 @@ public Producer(IEnumerable<KeyValuePair<string, object>> config)
         /// <summary>
         ///     Poll for callback events. You will not typically need
         ///     to call this method. Only call on producer instances 
-        ///     where background polling has been disabled.
+        ///     where background polling is not enabled.
```

Comment:
I meant that we should reference the constructor argument here, e.g.:
"where manual polling has been enabled".